### PR TITLE
[EN-44100] upgrade node version to 14

### PR DESCRIPTION
**Pull request by vickingan**

### CODE REVIEW

**Comment by vickingan** [comment]
Tested that `./bin/start.sh` still starts up the service and that `make test` works.

**Comment by langlk** [comment]
This looks good to me, but as you noted in slack, probably a good idea to also get a review from someone more familiar with this

File: package.json [hunk 1]
```diff
@@ -3,51 +3,55 @@
   "version": "0.1.89-SNAPSHOT",
   "private": true,
   "dependencies": {
-    "clarinet": "0.11.0",
     "async": "1.5.2",
-    "babel": "~5.8.29",
-    "body-parser": "1.14.2",
-    "bunyan": "1.5.1",
+    "babel": "^5.8.38",
```
Comment (MFry) [inline package.json:7]: Pretty much every dependency that has a `^` will need to have that removed, there might also be other issues we will need to check. (I just ran into one recently)

File: es6-lib/config/test.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ default {
 
   zk: ["localhost:2181"],
   log: {
-    level: 'critical',
+    level: 'fatal',
```
Comment (MFry) [inline es6-lib/config/test.js:10]: 🍖 bunyan changed 'critical' level to 'fatal'

File: package.json [hunk 3]
```diff
@@ -3,51 +3,54 @@
...
+    "negotiator": "0.6.1",
+    "node-expat": "2.3.18",
     "node-zookeeper-client": "0.2.2",
+    "npm": "7.6.3",
     "proj4": "2.3.12",
     "request": "2.88.2",
+    "set-value": "2.0.1",
     "shapefile": "rozap/shapefile",
-    "stompit": "0.23.0",
+    "srs": "socrata-platform/node-srs",
```
Comment (MFry) [inline package.json:40]: 🍖 We now have our own for of `node-srs`! Long live our own fork!

**Review by MFry — APPROVED** [review]
:shipit:

**Pull request merged**